Makefile

```diff
@@ -18,6 +18,7 @@ prep_lite: | $(TARGET_DIRS)
 	$(MAKE) libs
```

Comment:
I think you can revert all these changes?  bootrom gets built as part of programs, no reason to add a new target for it

Reply:
Well, I put it after the ucode because the default bootrom should have a ucode linked in right? So once the ucode is generated, I added in the bootrom generation.

Reply:
make sense!

Reply:
The default bootrom is actually committed to the repo, so we don't have to worry about it being generated.  The non-default bootrom should actually be generated as part of the simulation collateral, so part of the bp_tethered/Makefile.vcs flow.  That way, the user could specify by setting variables